tests/test_bulk_update.py

```diff
@@ -394,7 +396,7 @@ def test_auto_now(self):
 
         instance = AutoNowModel.objects.get()
         self.assertEqual(datetime(2019, 1, 1,  tzinfo=pytz.utc), instance.created)
-        self.assertEqual(instance.updated, date.today())
+        self.assertEqual(instance.updated, datetime.now(pytz.utc).date())
```

Comment:
Duplicate: What issue does this fix? How is it connected to table name issue of this PR?